packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -1548,6 +1548,48 @@ class ColorFilterLayer extends ContainerLayer {
   }
 }
 
+/// A composite layer that applies an [ImageFilter] to its children.
+class ImageFilterLayer extends ContainerLayer {
+  /// Creates a layer that applies an [ImageFilter] to its children.
+  ///
+  /// The [imageFilter] property must be non-null before the compositing phase
+  /// of the pipeline.
+  ImageFilterLayer({
+                     ui.ImageFilter imageFilter,
+                   }) : _imageFilter = imageFilter;
```

Comment:
trivial nit: flutter style is:

```dart
  ImageFilterLayer({
    ui.ImageFilter imageFilter,
  }) : _imageFilter = imageFilter;
```

Reply:
Thanks!  This is one of those annoying times when the Android Studio built-in formatting just refuses to do the right thing so I forced it.